verification/curator-service/ui/src/components/AutomatedSourceForm.tsx

```diff
@@ -0,0 +1,192 @@
+    },
+    allFormSections: {
+        marginBottom: '2em',
+        maxWidth: '60%',
+        paddingLeft: '2em',
+        paddingRight: '1em',
+        paddingTop: '0.5em',
+    },
+    statusMessage: {
+        marginTop: '2em',
+        maxWidth: '80%',
+    },
+    uploadBar: {
+        alignItems: 'center',
+        display: 'flex',
+        height: '4em',
+        marginTop: 'auto',
+    },
+    cancelButton: {
+        marginLeft: '1em',
+    },
+}));
+
+interface Props {
+    user: User;
+    onModalClose: () => void;
+}
+
+export interface AutomatedSourceFormValues {
+    url: string | null;
+    name: string | null;
+}
+
+const AutomatedSourceFormSchema = Yup.object().shape({
+    url: Yup.string().nullable().required('Required'),
+    name: Yup.string().nullable().required('Required'),
+});
+
+export default function AutomatedSourceForm(props: Props): JSX.Element {
+    const classes = useStyles();
+    const history = useHistory();
+    const [errorMessage, setErrorMessage] = React.useState('');
+
+    const createSource = async (
+        values: AutomatedSourceFormValues,
+    ): Promise<void> => {
+        const newSource = { name: values.name, origin: { url: values.url } };
+        try {
+            await axios.post('/api/sources', newSource);
+            setErrorMessage('');
+        } catch (e) {
+            setErrorMessage(JSON.stringify(e));
+            return;
+        }
+        // Navigate to sources after successful submit
+        history.push({
+            pathname: '/sources',
+        });
+    };
+
+    return (
+        <AppModal
+            title="New automated data source"
+            onModalClose={props.onModalClose}
+        >
+            <Formik
+                validationSchema={AutomatedSourceFormSchema}
+                validateOnChange={false}
+                initialValues={{ url: null, name: null }}
+                onSubmit={async (values): Promise<void> => {
+                    await createSource(values);
+                }}
+            >
+                {({ isSubmitting, submitForm }): JSX.Element => (
+                    <div className={classes.form}>
+                        <div className={classes.headerText}>
+                            <Typography data-testid="header-title" variant="h5">
+                                Provide details about the automated data source
+                            </Typography>
+                            <Typography
+                                className={classes.headerBlurb}
+                                data-testid="header-blurb"
+                                variant="body2"
+                            >
+                                Add new cases through automated ingestion from a
+                                data source. G.h List will check for updates
+                                every 15 minutes from this new source. After you
+                                create this source follow these instructions to
```

Comment:
Should this be a link to instructions?

Reply:
I'd lifted this from the mocks, which didn't have a link. It's likely a placeholder -- I'll remove this sentence until we have the instructions ready to rock.